option.go

```diff
@@ -355,6 +357,21 @@ func setupBuiltinFunctions(opts ...Option) []Option {
 		Func("string", func(v interface{}) string { return cast.ToString(v) }),
 		Func("int", func(v interface{}) int { return cast.ToInt(v) }),
 		Func("bool", func(v interface{}) bool { return cast.ToBool(v) }),
+		Func("compare", func(x, y interface{}, ignoreKeys ...string) bool {
+			diff := cmp.Diff(x, y, cmpopts.IgnoreMapEntries(func(key string, val interface{}) bool {
+				for _, ignore := range ignoreKeys {
+					if key == ignore {
+						return true
+					}
+				}
+				return false
+			}))
+
+			// FIXME: Debug output of diffs
```

Comment:
At this point, it was possible to refer to the debug of the book object, but because the operator did not exist, debug output could not be generated.

Reply:
Just a thought, how about adding `diff()` too?

I could use `diff()` with Dump runner, maybe for debugging `compare()`.

Reply:
https://github.com/k1LoW/runn/issues/112

It seems like a good idea to have a diff function once the above support is complete.

```
-----START TEST CONDITION-----
diff(steps[8].res.body, vars.wantBody, "Content-Length") != ""


├── diff(steps[8].res.body, vars.wantBody, "Content-Length")
｜         ├── steps[8].res.body => {"Content-Type":"application/json","freeform":"foo", "Content-Length": 81}
｜         ├── vars.wantBody => {"Content-Type":"application/json","freeform":"foo"}
｜         └── "Content-Length" => "Content-Length"
└──  (diff contents) => ""
-----END TEST CONDITION-----
```

Reply:
GREAT!!!!